pkg/ovs/openflow/interfaces.go

```diff
@@ -179,3 +190,13 @@ type ctBase struct {
 	ctTable uint8
 	ctZone  uint16
 }
+
+type IPRange struct {
+	MinIP net.IP
```

Comment:
MinIP to Start or StartIP
MaxIP to End or EndIP

Reply:
Updated.